src/App.jsx

```diff
@@ -1,45 +1,18 @@
 import React from 'react';
-import './App.scss';
-
-export const misha = {
-  name: 'Misha',
-  age: 37,
-  sex: 'm',
-  isMarried: true,
-  partnerName: 'Natasha',
-};
-
-export const olya = {
-  name: 'Olya',
-  sex: 'f',
-  isMarried: true,
-  partnerName: 'Maksym',
-};
-
-export const alex = {
-  name: 'Alex',
-  age: 25,
-  sex: 'm',
-  isMarried: false,
-};
+import { Person, misha, olya, alex } from './components/Person';
```

Comment:
you are importing people from Person component & pass at as props to that component, looks weird.
Leave array in App